src/ProgressBar.js

```diff
@@ -58,7 +52,14 @@ const propTypes = {
   label: PropTypes.node,
   srOnly: PropTypes.bool,
   striped: PropTypes.bool,
-  active: PropTypes.bool,
+  animated: PropTypes.bool,
+  bsPrefix: PropTypes.string,
+  /**
+   * Sets the background class of the progress bar.
+   *
+   * @type ('success'|'danger'|'warning'|'info')
+   */
+  variant: PropTypes.oneOf(['success','danger','warning','info']),
```

Comment:
We want to keep the propType here `string` so folks can extend with there own variants, the doclet `@type` above tho will show the "default" variants in the docs which is good.